diff -Naur ImageMagick-7.0.6-10.orig/magick.sh.in ImageMagick-7.0.6-10/magick.sh.in
--- ImageMagick-7.0.6-10.orig/magick.sh.in	2017-08-27 19:25:48.000000000 +0000
+++ ImageMagick-7.0.6-10/magick.sh.in	2018-02-11 12:58:19.922000000 +0000
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/pkgs/bash/bin/bash
 #
 #  Copyright 1999-2017 ImageMagick Studio LLC, a non-profit organization
 #  dedicated to making software imaging solutions freely available.
@@ -35,7 +35,7 @@
   echo "$@"
 fi
 env \
-  LD_LIBRARY_PATH="${top_builddir}/MagickCore/.libs:${top_builddir}/MagickWand/.libs${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}"" \
+  LD_LIBRARY_PATH="/usr/pkgs/glibc/lib:${top_builddir}/MagickCore/.libs:${top_builddir}/MagickWand/.libs${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}"" \
   MAGICK_CODER_MODULE_PATH="${MAGICK_CODER_MODULE_PATH}" \
   MAGICK_CONFIGURE_PATH="${MAGICK_CONFIGURE_BUILD_PATH}${DIRSEP}${MAGICK_CONFIGURE_SRC_PATH}" \
   MAGICK_FILTER_MODULE_PATH="${MAGICK_FILTER_MODULE_PATH}" \
diff -Naur ImageMagick-7.0.6-10.orig/winpath.sh ImageMagick-7.0.6-10/winpath.sh
--- ImageMagick-7.0.6-10.orig/winpath.sh	2017-08-27 19:25:48.000000000 +0000
+++ ImageMagick-7.0.6-10/winpath.sh	2018-02-11 12:58:57.338000000 +0000
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/pkgs/bash/bin/bash
 # Copyright (C) 1999-2016 ImageMagick Studio LLC
 # Copyright (C) 2003 - 2008 GraphicsMagick Group
 #
