diff -Naur fluxbox-1.3.7.orig/data/Makemodule.am fluxbox-1.3.7/data/Makemodule.am
--- fluxbox-1.3.7.orig/data/Makemodule.am	2015-02-08 10:44:45.333187008 +0000
+++ fluxbox-1.3.7/data/Makemodule.am	2018-02-10 21:15:15.932000000 +0000
@@ -53,7 +53,7 @@
 data/menu:
 	env MENUFILENAME=$@ \
 		PREFIX="${prefix}" \
-		MENUTITLE="Fluxbox-${VERSION}" \
+		MENUTITLE="Lombix 2018" \
 		INSTALL=Yes \
 		PATH="${PATH}:../util:../util/fbrun" \
 		sh util/fluxbox-generate_menu
diff -Naur fluxbox-1.3.7.orig/data/init.in fluxbox-1.3.7/data/init.in
--- fluxbox-1.3.7.orig/data/init.in	2015-02-08 10:44:45.333187008 +0000
+++ fluxbox-1.3.7/data/init.in	2018-02-10 21:15:15.936000000 +0000
@@ -1,7 +1,7 @@
 ! If you're looking for settings to configure, they won't be saved here until
 ! you change something in the fluxbox configuration menu.
 
-session.menuFile:	~/.@pkgprefix@fluxbox@pkgsuffix@/menu
-session.keyFile: ~/.@pkgprefix@fluxbox@pkgsuffix@/keys
+session.menuFile:	~/.config/@pkgprefix@fluxbox@pkgsuffix@/menu
+session.keyFile: ~/.config/@pkgprefix@fluxbox@pkgsuffix@/keys
 session.styleFile: @default_style@
 session.configVersion:	13
diff -Naur fluxbox-1.3.7.orig/src/FbCommands.cc fluxbox-1.3.7/src/FbCommands.cc
--- fluxbox-1.3.7.orig/src/FbCommands.cc	2015-02-08 10:44:45.353187009 +0000
+++ fluxbox-1.3.7/src/FbCommands.cc	2018-02-10 21:15:15.940000000 +0000
@@ -153,7 +153,7 @@
     // this process exits immediately, so we don't have to worry about memleaks
     const char *shell = getenv("SHELL");
     if (!shell)
-        shell = "/bin/sh";
+        shell = "/usr/pkgs/bash/bin/bash";
 
     setsid();
     execl(shell, shell, "-c", m_cmd.c_str(), static_cast<void*>(NULL));
diff -Naur fluxbox-1.3.7.orig/src/cli_options.cc fluxbox-1.3.7/src/cli_options.cc
--- fluxbox-1.3.7.orig/src/cli_options.cc	2015-02-08 10:44:45.377187009 +0000
+++ fluxbox-1.3.7/src/cli_options.cc	2018-02-10 21:15:15.943000000 +0000
@@ -48,7 +48,7 @@
         session_display.assign(env);
     }
 
-    string fname = std::string("~/.") + realProgramName("fluxbox");
+    string fname = std::string("~/.config/") + realProgramName("fluxbox");
     rc_path = FbTk::StringUtil::expandFilename(fname);
 
     if (!rc_path.empty()) {
diff -Naur fluxbox-1.3.7.orig/src/main.cc fluxbox-1.3.7/src/main.cc
--- fluxbox-1.3.7.orig/src/main.cc	2015-02-08 10:44:45.377187009 +0000
+++ fluxbox-1.3.7/src/main.cc	2018-02-10 21:15:15.946000000 +0000
@@ -270,7 +270,7 @@
         if (!restart_argument.empty()) {
             const char *shell = getenv("SHELL");
             if (!shell)
-                shell = "/bin/sh";
+                shell = "/usr/pkgs/bash/bin/bash";
 
             execlp(shell, shell, "-c", restart_argument.c_str(), (const char *) NULL);
             perror(restart_argument.c_str());
diff -Naur fluxbox-1.3.7.orig/util/fbrun/main.cc fluxbox-1.3.7/util/fbrun/main.cc
--- fluxbox-1.3.7.orig/util/fbrun/main.cc	2015-02-08 10:44:45.377187009 +0000
+++ fluxbox-1.3.7/util/fbrun/main.cc	2018-02-10 21:15:15.954000000 +0000
@@ -64,7 +64,7 @@
         "   -fg [color name]            Foreground text color"<<endl<<
         "   -bg [color name]            Background color"<<endl<<
         "   -na                         Disable antialias"<<endl<<
-        "   -hf [history file]          History file to load (default ~/.fluxbox/fbrun_history)"<<endl<<
+        "   -hf [history file]          History file to load (default ~/.config/fluxbox/fbrun_history)"<<endl<<
         "   -help                       Show this help"<<endl<<endl<<
         "Example: fbrun -fg black -bg white -text xterm -title \"run xterm\""<<endl;
 }
@@ -82,7 +82,7 @@
     string foreground("black");   // text color
     string background("white");   // text background color
     string display_name; // name of the display connection
-    string history_file("~/.fluxbox/fbrun_history"); // command history file
+    string history_file("~/.config/fluxbox/fbrun_history"); // command history file
     // parse arguments
     for (int i=1; i<argc; i++) {
         string arg = argv[i];
diff -Naur fluxbox-1.3.7.orig/util/fbsetbg fluxbox-1.3.7/util/fbsetbg
--- fluxbox-1.3.7.orig/util/fbsetbg	2015-02-08 10:44:45.377187009 +0000
+++ fluxbox-1.3.7/util/fbsetbg	2018-02-10 21:15:15.956000000 +0000
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/pkgs/bash/bin/bash
 #
 # Set wallpaper for fluxbox.
 #
@@ -45,11 +45,11 @@
 
 # The wallpapersetter is selected in this order
 wpsetters="${wpsetters:=Esetroot wmsetbg feh hsetroot chbg display qiv xv xsri xli xsetbg}" # broken icewmbg'
-lastwallpaper="${HOME}/.fluxbox/lastwallpaper"
+lastwallpaper="${HOME}/.config/fluxbox/lastwallpaper"
 
 
 WHOAMI=`whoami`
-[ "$WHOAMI" = root ] && PATH=/bin:/usr/bin/:/usr/local/bin:/usr/X11R6/bin
+[ "$WHOAMI" = root ] && PATH=/usr/bin:/usr/pkgs/X11/bin:/usr/pkgs/coreutils/bin:/usr/pkgs/util-linux/bin:/usr/pkgs/findutils/bin:/usr/pkgs/grep/bin:/usr/pkgs/sed/bin:/usr/pkgs/fluxbox/bin:/usr/pkgs/gawk/bin
 
 command="`basename \"$0\"`"
 
@@ -99,7 +99,7 @@
 
 Files:
 
-    ~/.fluxbox/lastwallpaper     In this file the wallpaper you set
+    ~/.config/fluxbox/lastwallpaper     In this file the wallpaper you set
                                  will be stored, for the -l option:
 Environment variables:
     wpsetters   Wallpapersetters to use.
@@ -123,7 +123,7 @@
    perl -pi -e 's,([^f]|^)bsetbg,$command,' filename
 
 2) If you want to set the wallpaper and you want $command to
-   remember the previous wallpaper put this in your ~/.fluxbox/overlay
+   remember the previous wallpaper put this in your ~/.config/fluxbox/overlay
 
    background: none
 
@@ -194,8 +194,8 @@
 fi
 
 # create directory and last wallpaper file 
-if [ ! -d  "$HOME/.fluxbox" ]; then
-	mkdir "$HOME/.fluxbox"
+if [ ! -d  "$HOME/.config/fluxbox" ]; then
+	mkdir "$HOME/.config/fluxbox"
     touch ${lastwallpaper}
 fi
 
diff -Naur fluxbox-1.3.7.orig/util/fluxbox-generate_menu.in fluxbox-1.3.7/util/fluxbox-generate_menu.in
--- fluxbox-1.3.7.orig/util/fluxbox-generate_menu.in	2015-02-08 10:44:45.377187009 +0000
+++ fluxbox-1.3.7/util/fluxbox-generate_menu.in	2018-02-10 21:19:44.161000000 +0000
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/pkgs/bash/bin/bash
 #
 # generate_menu for Fluxbox
 #
@@ -76,8 +76,8 @@
     -w  Homepage for console-browsers. Default is fluxbox.org
     -b  Favourite browser
     -m  Menu-title; default is "Fluxbox"
-    -o  Outputfile; default is ~/.@pkgprefix@fluxbox@pkgsuffix@/menu
-    -u  User sub-menu; default is ~/.@pkgprefix@fluxbox@pkgsuffix@/usermenu
+    -o  Outputfile; default is ~/.config/@pkgprefix@fluxbox@pkgsuffix@/menu
+    -u  User sub-menu; default is ~/.config/@pkgprefix@fluxbox@pkgsuffix@/usermenu
 
     -h  Display this help
     -a  Display the authors of this script
@@ -90,8 +90,8 @@
 
 
 Files:
-    ~/.@pkgprefix@fluxbox@pkgsuffix@/usermenu     Your own submenu which will be included in the menu
-    ~/.@pkgprefix@fluxbox@pkgsuffix@/menuconfig   rc file for fluxbox-generate_menu
+    ~/.config/@pkgprefix@fluxbox@pkgsuffix@/usermenu     Your own submenu which will be included in the menu
+    ~/.config/@pkgprefix@fluxbox@pkgsuffix@/menuconfig   rc file for fluxbox-generate_menu
 
 EOF
 }
@@ -345,7 +345,7 @@
     # $dnlamVERBOSE "# creating `date`" >> $ICONMAPPING
     # $dnlamVERBOSE "# using desktop files in $@" >> $ICONMAPPING
     # $dnlamVERBOSE "# searching for icons in `eval toSingleLine $OTHER_ICONPATHS`" >> $ICONMAPPING
-    # need to determine when to use .fluxbox/icons/$execname.xpm over those listed in iconmapping
+    # need to determine when to use .config/fluxbox/icons/$execname.xpm over those listed in iconmapping
     # $dnlamVERBOSE echo "createIconMapping: $@"
     for DIR in "$@" ; do
         if [ -d "$DIR" ]; then
@@ -558,7 +558,7 @@
 
 
 WHOAMI=`whoami`
-[ "$WHOAMI" = root ] && PATH=/bin:/usr/bin:/usr/local/bin:/usr/X11R6/bin
+#[ "$WHOAMI" = root ] && PATH=/usr/bin:/usr/pkgs/fluxbox/bin:/usr/pkgs/X11/bin:/usr/pkgs/coreutils/bin:/usr/pkgs/util-linux/bin:/usr/pkgs/findutils/bin:/usr/pgks/gawk/bin:/usr/pkgs/sed/bin:/usr/pkgs/diffutils/bin:/usr/pkgs/grep/bin
 
 # Check for Imlib2-support
 if @pkgprefix@fluxbox@pkgsuffix@@EXEEXT@ -info 2> /dev/null | grep -q "^IMLIB"; then
@@ -1220,10 +1220,10 @@
 esac
 
 # Set Defaults
-USERFLUXDIR="${HOME}/.@pkgprefix@fluxbox@pkgsuffix@"
+USERFLUXDIR="${HOME}/.config/@pkgprefix@fluxbox@pkgsuffix@"
 MENUFILENAME="${MENUFILENAME:=${USERFLUXDIR}/menu}"
-MENUTITLE="${MENUTITLE:=Fluxbox}"
-HOMEPAGE="${HOMEPAGE:=fluxbox.org}"
+MENUTITLE="${MENUTITLE:=Lombix 2018}"
+HOMEPAGE="${HOMEPAGE:=lombix.org}"
 USERMENU="${USERMENU:=${USERFLUXDIR}/usermenu}"
 MENUCONFIG="${MENUCONFIG:=${USERFLUXDIR}/menuconfig}"
 DOSUDO="no"
@@ -1258,7 +1258,7 @@
 # HOMEPAGE=fluxbox.org
 
 # location with your own menu-entries
-# USERMENU=~/.@pkgprefix@fluxbox@pkgsuffix@/usermenu
+# USERMENU=~/.config/@pkgprefix@fluxbox@pkgsuffix@/usermenu
 
 # Put the launcher you would like to use here
 # LAUNCHER=@pkgprefix@fbrun@pkgsuffix@@EXEEXT@
@@ -1485,7 +1485,7 @@
 
 # directory for the backgrounds
 if [ -z "$BACKGROUND_DIRS" ]; then
-    BACKGROUND_DIRS="${USERFLUXDIR}/backgrounds/:${PREFIX}/share/fluxbox/backgrounds/"
+    BACKGROUND_DIRS="${USERFLUXDIR}/backgrounds/:${PREFIX}/share/fluxbox/backgrounds/:/usr/share/fluxbox/backgrounds"
 fi
 
 # find the default terminal
@@ -1561,7 +1561,7 @@
 # If you read this it means you want to edit this file manually, so here
 # are some useful tips:
 #
-# - You can add your own menu-entries to ~/.@pkgprefix@fluxbox@pkgsuffix@/usermenu
+# - You can add your own menu-entries to ~/.config/@pkgprefix@fluxbox@pkgsuffix@/usermenu
 #
 # - If you miss apps please let me know and I will add them for the next
 #   release.
@@ -1570,8 +1570,8 @@
 #   makes things much more readable.
 #
 # - To prevent any other app from overwriting your menu
-#   you can change the menu name in ~/.@pkgprefix@fluxbox@pkgsuffix@/init to:
-#     session.menuFile: ~/.@pkgprefix@fluxbox@pkgsuffix@/my-menu
+#   you can change the menu name in ~/.config/@pkgprefix@fluxbox@pkgsuffix@/init to:
+#     session.menuFile: ~/.config/@pkgprefix@fluxbox@pkgsuffix@/my-menu
 
 EOF
 
@@ -1724,9 +1724,10 @@
 append_submenu "${OFFICEMENU}"
     normal_find xclock xcalc kcalc grisbi qbankmanager evolution
     find_it gcalc           append "[exec] (gcalc) {gcalc}" || \
-        find_it gnome-calculator append "[exec] (gcalc) {gnome-calculator}"
+        find_it gnome-calculator append "[exec] (gcalc) {gnome-calculator}"     
     find_it ical            append "[exec] (Calendar)   {ical}"
-
+    find_it galculator	 append "[exec] (galculator) {galculator}"
+    
     # older <=1.1.3 apparently have stuff like swriter, not sowriter
     for ext in s so oo xoo; do
         find_it ${ext}ffice2 && (
@@ -1827,11 +1828,11 @@
     append "[config] (${CONFIGUREMENU})"
 
     append_menu "[submenu] (${SYSTEMSTYLES}) {${STYLEMENUTITLE}}"
-        append "[stylesdir] (${PREFIX}/share/fluxbox/styles)"
+        append "[stylesdir] (/usr/share/fluxbox/styles)"
     append_menu_end
 
     append_menu "[submenu] (${USERSTYLES}) {${STYLEMENUTITLE}}"
-        append "[stylesdir] (~/.@pkgprefix@fluxbox@pkgsuffix@/styles)"
+        append "[stylesdir] (~/.config/@pkgprefix@fluxbox@pkgsuffix@/styles)"
     append_menu_end
 
     # Backgroundmenu
diff -Naur fluxbox-1.3.7.orig/util/fluxbox-update_configs.cc fluxbox-1.3.7/util/fluxbox-update_configs.cc
--- fluxbox-1.3.7.orig/util/fluxbox-update_configs.cc	2015-02-08 10:44:45.381187009 +0000
+++ fluxbox-1.3.7/util/fluxbox-update_configs.cc	2018-02-10 21:15:15.967000000 +0000
@@ -99,7 +99,7 @@
 void update_move_groups_entries_to_apps_file(FbTk::ResourceManager& rm,
         const FbTk::FbString& keyfilename, const FbTk::FbString& appsfilename) {
 
-    FbTk::Resource<string> rc_groupfile(rm, "~/.fluxbox/groups",
+    FbTk::Resource<string> rc_groupfile(rm, "~/.config/fluxbox/groups",
             "session.groupFile", "Session.GroupFile");
     string groupfilename = FbTk::StringUtil::expandFilename(*rc_groupfile);
     string whole_groupfile = read_file(groupfilename);
@@ -540,9 +540,9 @@
 int run_updates(int old_version, FbTk::ResourceManager &rm) {
     int new_version = old_version;
 
-    FbTk::Resource<string> rc_keyfile(rm, "~/.fluxbox/keys",
+    FbTk::Resource<string> rc_keyfile(rm, "~/.config/fluxbox/keys",
             "session.keyFile", "Session.KeyFile");
-    FbTk::Resource<string> rc_appsfile(rm, "~/.fluxbox/apps",
+    FbTk::Resource<string> rc_appsfile(rm, "~/.config/fluxbox/apps",
             "session.appsFile", "Session.AppsFile");
 
     string appsfilename = FbTk::StringUtil::expandFilename(*rc_appsfile);
@@ -600,7 +600,7 @@
     }
 
     if (rc_filename.empty())
-        rc_filename = FbTk::StringUtil::expandFilename("~/.fluxbox/init");
+        rc_filename = FbTk::StringUtil::expandFilename("~/.config/fluxbox/init");
 
     FbTk::ResourceManager resource_manager(rc_filename.c_str(),false);
     if (!resource_manager.load(rc_filename.c_str())) {
diff -Naur fluxbox-1.3.7.orig/util/startfluxbox.in fluxbox-1.3.7/util/startfluxbox.in
--- fluxbox-1.3.7.orig/util/startfluxbox.in	2015-02-08 10:44:45.381187009 +0000
+++ fluxbox-1.3.7/util/startfluxbox.in	2018-02-10 21:15:15.971000000 +0000
@@ -1,7 +1,7 @@
-#!/bin/sh
+#!/usr/pkgs/bash/bin/bash
 
 command="`basename \"$0\"`"
-fluxdir="$HOME/.@pkgprefix@fluxbox@pkgsuffix@"
+fluxdir="$HOME/.config/@pkgprefix@fluxbox@pkgsuffix@"
 startup="$fluxdir/startup"
 
 while [ $# -gt 0 ]; do
@@ -33,7 +33,7 @@
     fi
     if [ ! -r "$startup" ]; then
         ( cat << EOF
-#!/bin/sh
+#!/usr/pkgs/bash/bin/bash
 #
 # fluxbox startup-script:
 #
